src/core/lang.c

```diff
@@ -315,6 +315,15 @@ const uint8_t *lang_get_string(int group, int index)
         }
     }
 
+    if (group == 131) {
```

Comment:
I am unsure if group 131 is un-used by the original game. Otherwise, it will override some text.

Reply:
This is not how you're supposed to do this because,  as you mentioned, you may be replacing other strings.

Check my comment on Discord for the correct way to show a custom string

Reply:
Fixed.